lib/pf/activation.pm

```diff
@@ -591,17 +571,15 @@ sub set_status_verified_by_mac {
 sub activation_has_entry {
     my ($mac,$type) = @_;
     my ($status, $iter) = pf::dal::activation->search(
-        {
-            -where => {
-                mac => $mac,
-                type => $type,
-                status => $UNVERIFIED,
-                expiration => { ">=" => \['NOW()']},
-            },
-            -from => pf::dal::activation->find_from_tables(),
-            -columns => [\1],
-            -limit => 1,
-        }
+        -where => {
+            mac => $mac,
+            type => $type,
+            status => $UNVERIFIED,
+            expiration => { ">=" => \['NOW()']},
+        },
+        -from => pf::dal::activation->find_from_tables(),
```

Comment:
Not 100% sure, but I believe we said this shouldn't be used anymore because it should already do this behavior by default